.coveragerc

```diff
@@ -420,6 +420,9 @@ omit =
     homeassistant/components/mikrotik/*
     homeassistant/components/mill/climate.py
     homeassistant/components/mill/const.py
+    homeassistant/components/minecraft_server/__init__.py
+    homeassistant/components/minecraft_server/const.py
```

Comment:
I think you can remove `const.py` the coverage is nice with constants.

Reply:
I would like to keep it there. Unit testing a constants file doesn't sound right to me (although pytest might show 100% "coverage").